services/settings-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SettingsService.java

```diff
@@ -730,7 +729,7 @@ public APIGatewayProxyResponseEvent deleteAppConfig(Map<String, Object> event, C
             return new APIGatewayProxyResponseEvent().withHeaders(CORS).withStatusCode(200);
         }
 
-        long startTimeMillis = System.currentTimeMillis();
+        final long startTimeMillis = System.currentTimeMillis();
```

Comment:
Looks like we're collecting this data to LOG exec times to the CW LogGroup, but Lambda collects its own metrics and posts to CloudWatch by default. Do we need this?